ADWB_container/run.sh

```diff
@@ -0,0 +1,24 @@
+## build triplifier
+git clone --recurse-submodules https://github.com/MaastrichtU-CDS/triplifier-boot
+cd triplifier-boot
+docker run --rm -it \
+    -v $(pwd):/triplifier-boot \
+    -v $(pwd)/../build_triplifier_boot.sh:/triplifier-boot/build.sh \
+    --workdir /triplifier-boot \
+    maven:3.6.0-jdk-11-slim /bin/bash build.sh
+mv target/triplifier-boot-0.0.1-SNAPSHOT.jar ../triplifier-boot-0.0.1-SNAPSHOT.jar
```

Comment:
I need to use sudo to move the jar and also to delete the triplifier-boot dir. I tried to use the `--user` tag for `docker run`, but it didn't help. The output of `docker run` is owned by root. The easy, but not elegant, solution is to add sudo here XD.